### PR TITLE
Addf32_fast_f8-series into compute types.

### PR DESCRIPTION
**Pull request by mengzcai**
Addf32_fast_f8-series into compute types.

### CODE REVIEW

**Comment by mengzcai** [comment]
Project SWDEV-440166 is currently in progress and these types are required.
https://ontrack-internal.amd.com/browse/SWDEV-440166

File: library/include/hipblas.h [hunk 1]
```diff
@@ -572,6 +572,12 @@ typedef enum
     HIPBLAS_COMPUTE_64F_PEDANTIC = 8, /**< compute will be exactly 64-bit precision */
     HIPBLAS_COMPUTE_32I          = 9, /**< compute will be at least 32-bit integer precision */
     HIPBLAS_COMPUTE_32I_PEDANTIC = 10, /**< compute will be exactly 32-bit integer precision */
+    HIPBLAS_COMPUTE_32F_FAST_8F  = 100, /**< 32-bit input can use fp8 compute */
```
Comment (jichangjichang) [inline library/include/hipblas.h:575]: HIPBLAS_COMPUTE_32F_FAST_8F_FNUZ
Refer to https://github.com/ROCm/HIP/blob/9eb37e9947ff630d623e0172dc4150a6d640280f/include/hip/library_types.h#L62

File: library/include/hipblas.h [hunk 2]
```diff
@@ -572,6 +572,12 @@ typedef enum
...
+    HIPBLAS_COMPUTE_32F_FAST_8B  = 101, /**< 32-bit input can use bf8 compute */
+    HIPBLAS_COMPUTE_32F_FAST_8F8B
```
Comment (jichangjichang) [inline library/include/hipblas.h:577]: HIPBLAS_COMPUTE_32F_FAST_8F8BF_FNUZ

File: library/include/hipblas.h [hunk 3]
```diff
@@ -572,6 +572,12 @@ typedef enum
     HIPBLAS_COMPUTE_64F_PEDANTIC = 8, /**< compute will be exactly 64-bit precision */
     HIPBLAS_COMPUTE_32I          = 9, /**< compute will be at least 32-bit integer precision */
     HIPBLAS_COMPUTE_32I_PEDANTIC = 10, /**< compute will be exactly 32-bit integer precision */
+    HIPBLAS_COMPUTE_32F_FAST_8F  = 100, /**< 32-bit input can use fp8 compute */
+    HIPBLAS_COMPUTE_32F_FAST_8B  = 101, /**< 32-bit input can use bf8 compute */
```
Comment (jichangjichang) [inline library/include/hipblas.h:576]: HIPBLAS_COMPUTE_32F_FAST_8BF_FNUZ

File: library/include/hipblas.h [hunk 4]
```diff
@@ -568,10 +568,16 @@ typedef enum
...
     = 6, /**< 32-bit input can use tensor cores w/ TF32 compute. Only supported with cuBLAS backend currently */
-    HIPBLAS_COMPUTE_64F          = 7, /**< compute will be at least 64-bit precision */
-    HIPBLAS_COMPUTE_64F_PEDANTIC = 8, /**< compute will be exactly 64-bit precision */
-    HIPBLAS_COMPUTE_32I          = 9, /**< compute will be at least 32-bit integer precision */
-    HIPBLAS_COMPUTE_32I_PEDANTIC = 10, /**< compute will be exactly 32-bit integer precision */
+    HIPBLAS_COMPUTE_64F               = 7, /**< compute will be at least 64-bit precision */
+    HIPBLAS_COMPUTE_64F_PEDANTIC      = 8, /**< compute will be exactly 64-bit precision */
+    HIPBLAS_COMPUTE_32I               = 9, /**< compute will be at least 32-bit integer precision */
+    HIPBLAS_COMPUTE_32I_PEDANTIC      = 10, /**< compute will be exactly 32-bit integer precision */
+    HIPBLAS_COMPUTE_32F_FAST_8F_FNUZ  = 100, /**< 32-bit input can use fp8 compute */
```
Comment (jichangjichang) [inline library/include/hipblas.h:575]: add postfix "_EXT" for non-cublas parity cases

**Pull request closed without merging**